README.md

```diff
@@ -6,7 +6,7 @@ Switchtec Gen4 PAX is a variant of Microsemi's Switchtec PCIe switch product whi
 
 By default, an NVMe drive in the PAX fabric cannot be enumerated by the host. Using the bind operation provided by the Switchtec Gen4 PAX switch, such a VF can be bound to and enumerated by the host.
 
-Unlike some other SR-IOV devices such as NICs, NVMe drives need to be properly configured before they can be used. Generally, configuration includes creating an namespace and attaching the namespace to a specific VF (NVMe secondary controller). 
+Unlike some other SR-IOV devices such as NICs, NVMe drives need to be properly configured before they can be used. Generally, configuration includes creating a namespace and attaching the namespace to a specific VF (NVMe secondary controller), before bringing the VF online. 
 
```

Comment:
Virtual resource allocation and assignment are also common configurations for VFs.

Reply:
Updated.